packages/database/src/utils/__tests__/encryption.test.ts

```diff
@@ -0,0 +1,129 @@
+import { testData } from '../__fixtures__';
+import { createHash, decryptData, encryptData } from '../encryption';
+
+const { hashing, encryption, decryption } = testData;
+
+describe('Hashing', () => {
```

Comment:
Try to keep the description same as the function name

```suggestion
describe('createHash', () => {
```